packages/server-reason-react-ppx/test/test.re

```diff
@@ -265,6 +265,20 @@ let svg_2 = () => {
   );
 };
 
+let spellCheck_with_ppx = () => {
+  let div = <div spellCheck=false />;
+  assert_string(
+    ReactDOM.renderToStaticMarkup(div),
+    {|<div spellcheck="false"></div>|},
+  );
+};
+
+let spellCheck_without_ppx = () => {
+  let div =
+    React.createElement("div", ReactDOM.domProps(~spellCheck=false, ()), []);
+  assert_string(ReactDOM.renderToStaticMarkup(div), "<div ></div>");
```

Comment:
Notice how in this case, the attribute disappears, when it should be like the one above.

Reply:
spellCheck is a special case, and is not handled by server-reason-react.

Boolean attributes that are set to false aren't rendered on the resultant HTML, in this case spellCheck is an Enumerated attribute (I learn that today https://developer.mozilla.org/en-US/docs/Glossary/Enumerated)

I should add Enumerated as a Attribute variant.